chain/consensus/compute_state.go

```diff
@@ -135,6 +135,10 @@ func (t *TipSetExecutor) ApplyBlocks(ctx context.Context,
 			return xerrors.Errorf("running cron: %w", err)
 		}
 
+		if !ret.ExitCode.IsSuccess() {
+			return xerrors.Errorf("cron failed with exit code %d: %s", ret.ExitCode, ret.ActorErr.Error())
```

Comment:
nit for here and reward error handling: to maintain existing behavior you could end err string with %w and wrap the error instead of printing as string